core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -152,7 +152,26 @@ export class JestTask extends GulpTask<IJestConfig> {
     const pattern: string = path.join(srcRoot, '**/__snapshots__/*.snap');
     globby.sync(pattern).forEach(sourceFile => {
       const destination: string = sourceFile.replace(srcRoot, destRoot);
-      fsx.copySync(sourceFile, destination);
+      const jsDestination: string = destination.replace(/\.test\..+\.snap$/, '.test.js.snap');
```

Comment:
>const jsDestination: string = destination.replace(/\.test\..+\.snap$/, '.test.js.snap'); [](start = 6, length = 88)

Can you put { replace, check existing, and copy } into a function? So you can call the function four times instead of copying the code four times.

Reply:
Resolved.

---
In reply to: [172277005](https://github.com/Microsoft/web-build-tools/pull/465#discussion_r172277005) [](ancestors = 172277005)